dns/pfSense-pkg-bind9/files/usr/local/pkg/bind.inc

```diff
@@ -513,7 +510,10 @@ EOD;
 							}
 						}
 						for ($y = 0; $y < sizeof($zone['row']); $y++) {
-							$hostname = (preg_match("/(MX|NS)/", $zone['row'][$y]['hosttype']) ? "@" : $zone['row'][$y]['hostname']);
+							$hostname = $zone['row'][$y]['hostname'];
+							if (preg_match("/(MX|NS)/", $zone['row'][$y]['hosttype']) && trim($hostname) == false) {
```

Comment:
"==" is not a strict operator in PHP, so the string will be converted to a boolean value for the comparison, and any non-empty string converts to "true," otherwise "false" for an empty string. So, it is the case that ('' == false) is true, however, I see the prevailing style in this particular script is to test equality to an empty string, so I will change it.